modules/customize-certificates-replace-default-router.adoc

```diff
@@ -14,6 +14,13 @@ and CLI, will have encryption provided by specified certificate.
 
 * You must have a wildcard certificate and its private key,
 both in the PEM format, for use.
++
+[WARNING]
+====
+The new certificate *must* be signed by a public certificate authority already
+included in the CA bundle as provided by the container userspace.
+====
+
 * The certificate must have a `subjectAltName` extension of
 `*.apps.<clustername>.<domain>`.
 
```

Comment:
In essence, the wildcard cert SAN must match the cluster's ingress domain:
```
$ oc get ingress.config/cluster -o yaml
apiVersion: config.openshift.io/v1
kind: Ingress
metadata:
  name: cluster
spec:
  domain: apps.<clustername>.<domain>
status: {}
```

The SAN for the above ingress domain would be `*.apps.<clustername>.<domain>`